### PR TITLE
feat: added cairo connector

### PR DESCRIPTION
**Pull request by Lukasz2891**

### CODE REVIEW

File: packages/evm-connector/contracts/core/RedstoneConsumerBase.sol [hunk 1]
```diff
@@ -59,14 +59,14 @@ abstract contract RedstoneConsumerBase is CalldataExtractor {
     return RedstoneDefaultsLib.aggregateValues(values);
   }
 
-  /* ========== FUNCTIONS WITH IMPLEMENTATION (CAN NOT BE OVERRIDEN) ========== */
+  /* ========== FUNCTIONS WITH IMPLEMENTATION (CAN NOT BE overridden) ========== */
```
Comment (hatskier) [inline packages/evm-connector/contracts/core/RedstoneConsumerBase.sol:62]: I would stay with the upper case version here

File: packages/evm-connector/contracts/core/RedstoneConsumerBase.sol [hunk 2]
```diff
@@ -19,19 +19,19 @@ import "../libs/SignatureLib.sol";
 abstract contract RedstoneConsumerBase is CalldataExtractor {
   using SafeMath for uint256;
 
-  /* ========== VIRTUAL FUNCTIONS (MAY BE OVERRIDEN IN CHILD CONTRACTS) ========== */
+  /* ========== VIRTUAL FUNCTIONS (MAY BE overridden IN CHILD CONTRACTS) ========== */
```
Comment (hatskier) [inline packages/evm-connector/contracts/core/RedstoneConsumerBase.sol:22]: I would stay with the upper case version here

File: packages/evm-connector/contracts/core/CalldataExtractor.sol [hunk 3]
```diff
@@ -40,7 +40,7 @@ contract CalldataExtractor is RedstoneConstants {
       )
     }
     uint256 calldataNegativeOffset = unsignedMetadataByteSize
-      + UNSGINED_METADATA_BYTE_SIZE_BS
```
Comment (hatskier) [inline packages/evm-connector/contracts/core/CalldataExtractor.sol:43]: Can you please move these typo fixes to a separate Pull request?

File: packages/evm-connector/contracts/core/RedstoneConsumerBase.sol (hunk 1 above)
Comment (Lukasz2891) [reply packages/evm-connector/contracts/core/RedstoneConsumerBase.sol:62]: it will be replaced :)

File: packages/evm-connector/contracts/core/CalldataExtractor.sol (hunk 3 above)
Comment (Lukasz2891) [reply packages/evm-connector/contracts/core/CalldataExtractor.sol:43]: yes

File: packages/cache-service/src/data-packages/data-packages.service.ts [hunk 4]
```diff
@@ -188,6 +188,9 @@ export class DataPackagesService {
       requestParams
     );
 
+    console.log("XXX");
```
Comment (Lukasz2891) [inline packages/cache-service/src/data-packages/data-packages.service.ts:191]: wrr :)

**Review by hatskier — APPROVED** [review]
